centreon/www/front_src/src/Resources/Details/tabs/Graph/ChartGraph.tsx

```diff
@@ -86,44 +91,56 @@ const ChartGraph = ({
 
   const rest = areaThresholdLines ? { shapeLines: areaThresholdLines } : {};
 
-  return (
-    <>
-      <FederatedComponent
-        path="/anomaly-detection/enableThresholdLines"
-        styleMenuSkeleton={{ height: 0, width: 0 }}
-        type={resource?.type}
-        getShapeLines={getShapeLines}
-      />
-      <LineChart
-        loading={isFetching || isLoading || !data}
-        annotationEvent={{ data: timeLineData }}
-        containerStyle={classes.container}
-        getRef={getRef}
-        data={data}
-        end={graphTimeParameters?.end}
-        height={280}
-        legend={{ mode: 'grid', placement: 'bottom' }}
-        lineStyle={{ lineWidth: 1 }}
-        header={{ extraComponent: graphActions }}
-        tooltip={{
-          renderComponent: ({
-            data,
-            hideTooltip
-          }: TooltipData): JSX.Element => (
-            <Comment
-              commentDate={data}
-              hideAddCommentTooltip={hideTooltip}
-              resource={resource}
-            />
-          )
-        }}
-        start={graphTimeParameters?.start}
-        timeShiftZones={{ enable: true, getInterval }}
-        zoomPreview={{ enable: true, getInterval }}
-        {...rest}
+  const metricsCount = data?.metrics?.length ?? 0;
+  const displayMetricsGraphCapMessage = metricsCount > graphsCapNumber;
+
+  const graphComponent = displayMetricsGraphCapMessage
+    ? (
+      <TooManyElementsCard
```

Comment:
I don't think it's relevant on Resources status. Because graphs are auto-generated and the user don't easily manage them (they can but it's not handy). So if we display this message on auto-generated graphs, it's like putting a branch in our own wheels.
But this stuff is more meaningful on dashboard